pkg/auth/providers/azure/azure_provider_test.go

```diff
@@ -222,9 +354,13 @@ func TestTransformToAuthProvider(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			authProvider, err := provider.TransformToAuthProvider(test.authConfig)
 			assert.NoError(t, err)
-			url, ok := authProvider["redirectUrl"].(string)
-			assert.True(t, ok)
-			assert.Equal(t, test.expectedRedirectUrl, url)
+
+			// check the expected length, so if the keys change we can update the test
+			assert.Len(t, authProvider, len(test.expectedAuthProvider))
```

Comment:
Wouldn't `assert.Equal(t, test.expectedAuthProvider, authProvider)` do this?

Reply:
Done f1ef055